### PR TITLE
Create project structure

### LINKED ISSUE #7: [FEATURE] Create project structure
## Problem to solve

[//]: # "A clear and concise description of what the problem is"
Project need a good folder structure

## Further details

[//]: # "A clear and concise description of what you want to be documented."
Create folder structure for project

## Additional context

[//]: # "Add any other context or screenshots about the feature request here."

### PR DESCRIPTION
**Pull request by nvti**
## What does this MR do?

[//]: # "Briefly describe what this MR is about"
Prepare resource for working

## Related issues

[//]: # "Link related issues below. Insert the issue link or reference after the word `Closes` if merging this should automatically close it."
Closes #7 

## Author's checklist

- [x] Link related issues. If no related issue, create issue first.
- [x] Target to right branch
- [x] Set milestone
- [x] Assign to a reviewer
- [x] Apply label.

## Review checklist

- [x] Former merge request
- [x] Right milestone
- [x] Code/Document
  - [x] Follow [coding convention]()
  - [x] Follow [Documentation Guidelines]()
  - [x] Clarity
  - [x] Completeness
- [x] This merge request is not conflict

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,43 +7,42 @@
 
 ## Table of contents
```
Comment (honghanhh) [inline README.md:8]: Why remove the dot at the end of each item?

File: README.md [hunk 2]
```diff
@@ -7,43 +7,42 @@
...
-    - [ ] Final project poster session.
+    - [ ] [Final project poster session](Week%2011/Final%20project/README.md)
 
 ## Contributors:
 
-- [Viet-Tien](https://github.com/tiena2cva)
-- [honghanhh](https://github.com/honghanhh)
-
+- 🐔 [Viet-Tien](https://github.com/tiena2cva)
+- 🐮 [honghanhh](https://github.com/honghanhh)
```
Comment (honghanhh) [inline README.md]:
```suggestion
- 🐮 [@honghanhh](https://github.com/honghanhh)
```

File: README.md [hunk 3]
```diff
@@ -7,43 +7,42 @@
...
 - __Week 11__:
-    - [ ] Final project poster session.
+    - [ ] [Final project poster session](Week%2011/Final%20project/README.md)
 
 ## Contributors:
 
-- [Viet-Tien](https://github.com/tiena2cva)
-- [honghanhh](https://github.com/honghanhh)
-
+- 🐔 [Viet-Tien](https://github.com/tiena2cva)
```
Comment (honghanhh) [inline README.md]:
```suggestion
- 🐔 [@tiena2cva](https://github.com/tiena2cva)
```

File: README.md (hunk 1 above)
Comment (nvti) [reply README.md:8]: Because it is link now. I think the links end without dot. Right?

**Review by honghanhh — APPROVED** [review]

**Pull request merged**